relay-event-normalization/src/nel.rs

```diff
@@ -0,0 +1,77 @@
+//! Contains normalization for NEL reports.
+
+use chrono::{DateTime, Duration, Utc};
+use relay_event_schema::protocol::{
+    Contexts, Event, HeaderName, HeaderValue, Headers, LogEntry, NelContext, NetworkReportRaw,
+    PairList, Request, ResponseContext, Timestamp,
+};
+use relay_protocol::Annotated;
+
+/// Normalizes the event using the provided [`NetworkReportRaw`].
+pub fn normalize(event: &mut Event, nel: Annotated<NetworkReportRaw>) {
```

Comment:
I would move this function to `relay-server`, just to consistent with other cases in which we copy information from one envelope item to another. See for example: https://github.com/getsentry/relay/blob/4ed7d5fcec0caa3ca5db9f616527dcd612bfd5e4/relay-server/src/utils/unreal.rs#L266

Reply:
makes sense to me, moved this into `utils` in `relay-server`